### PR TITLE
Make ^10.0 the default constraint for Drupal core

### LINKED ISSUE #711: Make ^10.0 the default constraint for Drupal core, update from ^9.0
Drupal 9 is going away. It can remain in the test run, but should be considered the edge case and no longer the default.

### PR DESCRIPTION
**Pull request by mglaman**
Fixes #711

### CODE REVIEW

File: src/Rules/Deprecations/SymfonyCmfRouteObjectInterfaceConstantsRule.php [hunk 1]
```diff
@@ -26,6 +26,9 @@ public function getNodeType(): string
 
     public function processNode(Node $node, Scope $scope): array
     {
+        if (!class_exists(SymfonyRouteObjectInterface::class)) {
+            return [];
+        }
```
Comment (mglaman) [inline src/Rules/Deprecations/SymfonyCmfRouteObjectInterfaceConstantsRule.php:31]: This is causing `SymfonyCmfRouteObjectInterfaceConstantsRuleTest` to fail on ^9.0 and I'm not sure why.
